packages/wrangler/e2e/util.ts

```diff
@@ -0,0 +1,35 @@
+import assert from "assert";
```

Comment:
We have been avoiding the generic `utils` file naming in the codebase, is there a more explicit name we can use such as `e2e-setup-helpers` or `e2e-runner-helpers`